pkg/curatedpackages/packageinstaller.go

```diff
@@ -49,6 +53,18 @@ func newPackageController(installer ChartInstaller, runner KubectlRunner, spec *
 
 func (pi *Installer) InstallCuratedPackages(ctx context.Context) error {
 	PrintLicense()
+
+	// If cert-manager does not exist, instruct users to follow instructions in
+	// PrintCertManagerDoesNotExistMsg to install packages manually.
+	// Note although we passed in a namespace parameter in the kubectl command, the GetResource command will be
+	// performed in all namespaces since CRDs are not bounded by namespaces.
+	kubeConfig := kubeconfig.FromClusterName(pi.spec.Cluster.Name)
+	certManagerExists, _ := pi.kubectl.GetResource(ctx, "crd", "certificates.cert-manager.io", kubeConfig, constants.CertManagerNamespace)
+	if !certManagerExists {
+		PrintCertManagerDoesNotExistMsg()
+		return errors.New("cert-manager is not present in the cluster")
+	}
```

Comment:
This block of code is duplicated, could we, for example, extract it to a separate method?